bluepyopt/ephys/models.py

```diff
@@ -315,20 +316,42 @@ def create_hoc(self, param_values,
                 replace_axon += '\n'
                 replace_axon += morph_modifier_hoc
 
-        ret = create_hoc.create_hoc(mechs=self.mechanisms,
-                                    parameters=self.params.values(),
-                                    morphology=morphology,
-                                    ignored_globals=ignored_globals,
-                                    replace_axon=replace_axon,
-                                    template_name=template_name,
-                                    template_filename=template,
-                                    template_dir=template_dir,
-                                    disable_banner=disable_banner)
+        ret = sim_desc_creator(mechs=self.mechanisms,
```

Comment:
what if sim_desc_creator is None?

Reply:
Then the program is ill-posed as `_create_sim_desc` is a method meant only for use internally within the `CellModel` class. Would you like to capture that differently?

Reply:
Maybe it would be useful to raise a verbose exception in that case?

Reply:
Should be covered with e88168f.